doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1168,21 +1168,21 @@ You should expect JSON output and a 200 ("OK") response in most cases. If you re
 Delete Dataset Draft
 ~~~~~~~~~~~~~~~~~~~~
 
-Deletes the draft version of dataset ``$ID``. Only the draft version can be deleted:
+Deletes the draft version of dataset ``$PERSISTENT_IDENTIFIER``. Only the draft version can be deleted:
 
 .. code-block:: bash
 
   export API_TOKEN=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
   export SERVER_URL=https://demo.dataverse.org
-  export ID=24
+  export PERSISTENT_IDENTIFIER=doi:10.5072/FK2/BCCP9Z
 
-  curl -H "X-Dataverse-key: $API_TOKEN" -X DELETE $SERVER_URL/api/datasets/$ID/versions/:draft
+  curl -H "X-Dataverse-key: $API_TOKEN" -X DELETE $SERVER_URL/api/datasets/:persistentId/?persistentId=$PERSISTENT_IDENTIFIER
```

Comment:
This doesn't seem right. What was `versions` removed? Why was `:draft` removed?

Reply:
I don't know :). But this is the endpoint that works for me on ridagop.net  (dataverse: v. 5.12.1 build 1122-cf90431)  where I do not have any published datasets